modules/gotg/README.md

```diff
@@ -1,16 +1,16 @@
 # Bigby Presents: Glory of the Giants
 
-**A Manual of Mammoth Marvels**
+> A Manual of Mammoth Marvels
 
 Bigby the Great recounts his journeys into the history, myth, and society of giants in this guide to their realms in the worlds of Dungeons & Dragons. Here Bigby and the demigod Diancastra, child of the giants’ All-Father, Annam, unveil the secrets of Annam’s mighty descendants. Players will find a wealth of giant-themed character options: a subclass for barbarians, two backgrounds related to giants and their runic magic, and feat options that evoke the strength and primal magic of the giants. Dungeon Masters will gain inspiration for bringing giants to life, from maps of giant-built sites and big new monsters to tremendous treasures and intriguing trinkets in a giant’s bag! Let Bigby and Diancastra teach you everything about the iconic giants of D&D:
 
-- Delves into unrevealed giant lore, including an overview of the hierarchical structure of the ordning and giants’ religion, and ideas about giants’ organizations and societies across the multiverse.
-- Add giant-themed player character options with **1 new barbarian subclass**, **The Path of the Giant** for use in the **D&D Beyond character creator**
-- Explore the vast world of giants with **2 new backgrounds**, and **8 new feats to unleash runic magic and wield elemental power**
-- Discover a wondrous collection of **over 30 magic items**, including **3 illustrious artifacts**.
-- Encounter **over 70 new monsters and other enormous creatures** to face in the **D&D Beyond Encounter Builder**.
-- Offers a plethora of tools for Dungeon Masters including **lair maps**, **detailed lore**, **adventure hooks**, **encounter tables**, **treasures**, **and giant roleplaying inspiration**.
+* Delves into unrevealed giant lore, including an overview of the hierarchical structure of the ordning and giants’ religion, and ideas about giants’ organizations and societies across the multiverse.
+* Explore the vast world of giants with **2 new backgrounds**, and **8 new feats to unleash runic magic and wield elemental power**
+* Discover a wondrous collection of **over 30 magic items**, including **3 illustrious artifacts**.
+* Offers a plethora of tools for Dungeon Masters including **lair maps**, **detailed lore**, **adventure hooks**, **encounter tables**, **treasures**, **and giant roleplaying inspiration**.
+
+---
```

Comment:
There's a few places where the automated cleaning strips out things which could be relevant, like the "1 new barbarian subclass" here. It would be very nasty to make the automation only strip out _parts_ of lines, like the "for use in the D&D Beyond character creator" at the end of that line, so I don't think it's really worth the effort to address... just noting it for posterity.